accommodation/views.py

```diff
@@ -0,0 +1,95 @@
+        try:
+            checkin   = request.GET.get('checkin')
+            checkout  = request.GET.get('checkout')
+            guests    = request.GET.get('guests')
+            entire    = request.GET.get('entire') 
+            private   = request.GET.get('private') 
+            hotel     = request.GET.get('hotel')
+            shared    = request.GET.get('shared')
+            price_min = request.GET.get('min')
+            price_max = request.GET.get('max')
+
+            calendar_condition = Q()
+            if (checkin and checkout):
+                calendar_condition.add((
+                    Q(unavailabledate__start_date__gt= datetime.strptime(checkout, "%Y-%m-%d").date()) &
+                    Q(unavailabledate__end_date__lt=datetime.strptime(checkin, "%Y-%m-%d").date())
+                ), Q.OR)
+
+            q = Q()
+            guests_condition = Q()
+            if guests:
+                guests_condition = (Q(max_capacity__gt=guests))
+                q.add(guests_condition, Q.AND)
+
+            room_condition = Q()
+
+            if entire == 'true':
+                entire_condition  = Q(category__name='집 전체')
+                room_condition.add(entire_condition, Q.OR)
+
+            if private == 'true':
+                private_condition = Q(category__name='개인실')
+                room_condition.add(private_condition, Q.OR)
+
+            if shared == 'true':
+                shared_condition  = Q(category__name='다인실')
+                room_condition.add(shared_condition, Q.OR)
+
+            if hotel == 'true':
+                hotel_condition   = Q(category__name='호텔 객실')
+                room_condition.add(hotel_condition, Q.OR)
+                
+            q.add(room_condition, Q.AND)
+            
+            price_condition = Q()
+            if (price_min and price_max):
+                price_condition = (Q(price__lt=price_max) & Q(price__gt=price_min))
+                q.add(price_condition, Q.AND)
+            
+            accommodations = Accommodation.objects.prefetch_related('category').exclude(calendar_condition).filter(q)
+
+            data = [
+                {
+                'id'       : accommodation.id,
+                'img'      : [image.image_url for image in accommodation.image_set.all()],
+                'location' : '강남구 ' + accommodation.category.name,
+                'title'    : accommodation.title,
+                'MaxNum'   : accommodation.max_capacity,
+                'grade'    : str(round(mean(accommodation.review_set.all().aggregate(
+                    Avg('clean_rate'),
+                    Avg('communication_rate'),
+                    Avg('checkin_rate'),
+                    Avg('accuracy_rate'),
+                    Avg('location_rate'),
+                    Avg('value_rate')).values()), 2)) 
+                    if accommodation.review_set.all().exists() else '0',
+                'gradeNum' : accommodation.review_set.all().count(),
```

Comment:
```suggestion
                'gradeNum' : accommodation.review_set.count(),
```
마찬가지로 `all()`없이도 동작합니다 👍